source/fundamentals/data-formats/serialization.txt

```diff
@@ -0,0 +1,166 @@
+.. _csharp-serialization:
+
+=============
+Serialization
+=============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn about serialization with the {+driver-long+}.
+Serialization is the process of mapping a C# object to and from a BSON document.
+
+Serializer Registry
+-------------------
+
+The serializer registry is an implementation of ``IBsonSerializerRegistry``
+that contains all registered serializers. You can access the serializer registry
+with the ``SerializerRegistry`` property of the ``BsonSerializer``
+class as follows:
+
+.. code-block:: csharp
+
+   var intSerializer = BsonSerializer.SerializerRegistry.GetSerializer<int>();
+
+.. important::
+
+   The serializer registry is a global registry. This means that you cannot use
+   multiple registries in a single application.
+
+Serialization Provider
+----------------------
+
+A serialization provider allows you to provide serializers to your
+serialization registry. The serialization provider should be registered as early
```

Comment:
I: I don't fully understand what it means to provide a serialization to the registry. Just to add it by calling a method?

Reply:
After some discussion with engineering, we determined that we should remove the serialization provider section since it's more of an internal mechanism that doesn't need to be surfaced to the user.